packages/wd-base/src/_typography.scss

```diff
@@ -0,0 +1,123 @@
+/* ==========================================================================
+   Typography
+   ========================================================================== */
+
+/* Font Faces
+   ========================================================================== */
+$font-primary: "Circular", sans-serif !default;
+$font-secondary: "Open Sans", sans-serif !default;
+$font-mono: "Roboto Mono", sans-serif !default;
+
+
+// color
+$color-black: #0e141a;
+
+// font weight
+$display: 900;
+$emphasis: 500;
+$body: normal;
+
+// font family & color helper mixin
+@mixin font-primary($color-variable) {
+  font-family: $font-primary;
+  color: $color-variable;
+}
+
+
+/* Font Mixins
+   ========================================================================== */
+@mixin wd-typography-large($weight-variable) {
+  font-weight: $weight-variable;
```

Comment:
if you want to have multiple `weights` you should create another class just for that. it'd be easier to handle and you'll have less classes

Reply:
Regarding this, it turns out that the Zeplin Vitor made splits the styles into three: Display, Emphasis, and Body. However, the only difference between these styles is the weights. Display is 900, Emphasis is 500, and Body is 400/normal. Each of those 'styles' has 12 sizes.

I figured it would make sense to say `@include wd-typography-large($display)` when writing CSS, as well as having one simple class for a particular combination of typeface settings. With multiple classes, I imagine it might feel messy after awhile or be difficult to remember everything to include. This allows us to keep every aspect of a typeface in one mixin, so that it can be included in CSS for components really easily, and we don't have to worry about adding a separate line for font-weight or remembering another family of typography-related classes just to customize the style. It's more classes to have one for each one, but I like the simplicity (only calling one class instead of two to define a typeface, and having everything in one mixin in case we want to include the typeface in the component CSS)

What do you think? Or would you rather I make them separate?